contracts/contracts/harvest/OETHHarvesterSimple.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Governable } from "../governance/Governable.sol";
+import { IStrategy } from "../interfaces/IStrategy.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract OETHHarvesterSimple is Governable {
+    ////////////////////////////////////////////////////
+    /// --- STORAGE
+    ////////////////////////////////////////////////////
+    address public strategist;
+    mapping(address => bool) public supportedStrategies;
+
+    ////////////////////////////////////////////////////
+    /// --- EVENTS
+    ////////////////////////////////////////////////////
+    event Harvested(address token, uint256 amount);
+    event StrategistUpdated(address strategist);
+    event SupportedStrategyUpdated(address strategy, bool status);
+
+    ////////////////////////////////////////////////////
+    /// --- MODIFIERS
+    ////////////////////////////////////////////////////
+    modifier onlyStrategist() {
```

Comment:
this should be renamed to `onlyStrategistOrGovernor`

Reply:
Actually you could further simplify this contract by using `Strategizable`. This way any logic setting strategist won't need to be part of this contract

Reply:
Migrated to `Strategizable` in this commit:  [639e7ae](https://github.com/OriginProtocol/origin-dollar/pull/2333/commits/639e7aeb82f62add2003cf57a3fbe437cf362de5).